app/code/Magento/InventorySalesAlert/Api/ProductIsSalableInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesAlert\Api;
```

Comment:
More related name is `InventoryProductAlert` (now we have `ProductAlert` module but not `SalesAlert`)
NAmespace should be `Magento\ProductAlert\Model\...`